.github/workflows/build/Dockerfile.ubuntu-2004

```diff
@@ -76,3 +76,7 @@ RUN gem install --no-document fpm -v 1.13.1
 
 RUN apt-get -y autoremove \
     && rm -rf /var/lib/apt/lists/*
+
+COPY . /indy-node
+WORKDIR /indy-node
+RUN pip install .[tests]
```

Comment:
@pSchlarb, What is the impact of this on the `build_release` and `build_3rd_party_dependencies` jobs?  The build environment is not clean since it already has a code of the source code on it.

With the new `build-image` and `remove_test_image` process is there a possibility a `node-build` image might not exist by the time the `build_release` and `build_3rd_party_dependencies` jobs are run?

Reply:
Good  catch totally overlooked these.
I think there is no impact on them, as each Job starts its own container and they would run in the container for the actual commit. 
Will change them to use the code already present in the container.